mautic/mautic
# Update db_server_version on existing Mautic instances

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | 3.2
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | N/A
| Related user documentation PR URL      | N/A
| Related developer documentation PR URL | N/A
| Issue(s) addressed                     | Fixes #9200




#### Description:

Builds further on https://github.com/mautic/mautic/pull/9437, which only applies to **new** Mautic installations. This PR proactively fixes the `db_server_version` for existing instances. 



#### Steps to test this PR:
1. Ensure that you have a Mautic version from Git, LOWER THAN 3.2.0 installed (3.0.0 until 3.1.2). Example how to do this by cloning from Git:
```
git clone https://github.com/mautic/mautic.git mautic-temp
git checkout 3.1.2
(install Mautic now)
```
2. Open Configuration in the UI and click "Save and Close"
3. Open `app/config/local.php`. You should see a `db_server_version` parameter which should be lower than 5.7:

```PHP
<?php
$parameters = array(
    ....

    'db_server_version' => '5.5.5-10.2.34-MariaDB-1:10.2.34+maria~bionic-log',
    
    ....
);
```
4. Checkout this PR. Assuming you have the GitHub CLI installed, run `gh pr checkout 9458`
5. Run `bin/console doctrine:migrations:execute --up 20201130201631`
6. Open `app/config/local.php`. `db_server_version` should now equal 5.7.
